README.md

```diff
@@ -1,134 +1,122 @@
-
-When there are objections, notify the Solid Panels and Stakeholders about the final proposal with an invitation to vote. If there are options, detail them along with the implications of the options. The final proposal needs to be open for one week to give others a chance to vote. To show your support for a proposal write +1. To show your disapproval for a proposal write -1 along with a detailed reason for the disapproval and a suggestion for a preferred alternative. To abstain type 0 when you have no opinion. If you do not vote by the end of the week it will be assumed that you abstain. At the end of the week anyone may publish an overview of the vote results per Solid Panel and Stakeholder.
+A draft proposal often involves public discussion before being formally presented for review. Stakeholders and Panels may provide feedback on draft proposals.
 
 ### Reviewing proposals
 
 Candidate proposals to the [Solid Specification](https://github.com/solid/specification) submitted for review go through an editorial process before they are accepted.
-Candidate Proposals to change the Solid Specification must be submitted for editorial review before they are accepted, along with the results of any votes taken.
+
+The [Solid Editors](https://github.com/orgs/solid/teams/editors) [maintain a monthly development cycle](solid-editors-tr-process.md) in which they select proposals and issues from the open issues and proposals for consideration. 
+
+The Editors typically meet weekly, with an agenda put forward by the Solid Director with the help of other Editors and the Release Manager. In the interest of transparency, these meetings are to be recorded and made public within two days of the meeting.
 
 To help broad consensus form, it is suggested that each proposal be brought to the attention of the Editors at some defined contact points during its life cycle. These contact points are:
-1. When a reason to change or extend the specification is first encountered, an [issue is opened](https://github.com/solid/specification/issues/new). The issue is then used for further communication with the Editors.
+1. When a reason to change or extend the specification is first encountered, an [issue can be created](https://github.com/solid/specification/issues/new). The issue can then be used for further communication with the Editors.
 1. Panels may create or adopt issues, and Panels should notify Editors that such discussion has started.
 1. When a Panel reaches rough consensus on an issue, the Panel should notify the Editors, indicating in informal text what the consensus entails and who participated in forming it.
 1. When a Panel starts drafting the text of a proposal, the Panel should notify the Editors.
```

Comment:
How does one notify the editors?

Reply:
I assume through github tags? I didn't want to change the old text, which was unspecific in this regard.